tests/library/reduce_test.py

```diff
@@ -1,35 +1,82 @@
 # Copyright 2019-2021 ETH Zurich and the DaCe authors. All rights reserved.
-import dace
 import numpy as np
 import pytest
+
+import dace
 import dace.libraries.standard as std
+from dace import SDFG, Memlet
+
+C_in, C_out, H, K, N, W = (dace.symbol(s, dace.int64) for s in ('C_in', 'C_out', 'H', 'K', 'N', 'W'))
+
+
+def make_sdfg():
+    g = SDFG('prog')
+    g.add_array('A', (N, 1, 1, C_in, C_out), dace.float32,
+                strides=(C_in * C_out, C_in * C_out, C_in * C_out, C_out, 1))
+    g.add_array('C', (N, H, W, C_out), dace.float32,
+                strides=(C_out * H * W, C_out * W, C_out, 1))
+
+    st0 = g.add_state('st0', is_start_block=True)
+    st = st0
+
+    A = st.add_access('A')
+    C = st.add_access('C')
+    R = st.add_reduce('lambda x, y: x + y', [1, 2, 3], 0)
+    st.add_memlet_path(A, R, memlet=Memlet(expr='A[0:N, 0, 0, 0:C_in, 0:C_out]'))
+    st.add_memlet_path(R, C, memlet=Memlet(expr='C[0:N, 5, 5, 0:C_out]'))
```

Comment:
why use add_memlet_path for a path of length 1? You can just use `add_edge` or the shorthand for no connectors: `add_nedge`

Reply:
> why use add_memlet_path for a path of length 1?

Mostly because that was better (or at all) documented. Replaced now with `add_nedge()`, will also change this way for the other and future PRs.